data/schemas/node-type-genspec.schema

```diff
@@ -39,7 +39,7 @@
     },
 
     "data_type": {
-      "enum": [ "boolean", "int", "byte", "string", "float", "rgb", "direction-vector", "error", "location", "timestamp" ]
+      "enum": [ "boolean", "int", "byte", "string", "float", "rgb", "direction-vector", "error", "location", "timestamp", "key-value" ]
```

Comment:
The key-value type has an union in it, that can assume all the values that we need.
Still, should we make the distinction?